package.json

```diff
@@ -19,11 +20,17 @@
     "react-dom": "16.8.1",
     "react-google-maps": "9.4.5",
     "react-jvectormap": "0.0.6",
+    "react-redux": "7.0.3",
     "react-router-dom": "4.3.1",
     "react-scripts": "2.1.5",
     "react-swipeable-views": "0.13.1",
     "react-table": "6.9.2",
-    "react-tagsinput": "3.19.0"
+    "react-tagsinput": "3.19.0",
+    "redux": "4.0.1",
+    "redux-devtools-extension": "^2.13.8",
+    "redux-logger": "^3.0.6",
+    "redux-saga": "^1.0.2",
+    "redux-thunk": "^2.3.0"
```

Comment:
you have clear knowledge why this library is important? if not let me know.

Reply:
i don't, but i searched some information about it, and i found that his library allows you to write action creators that return a function instead of an action, i decide implement that because i take my example of redux of platzi.

Reply:
yeah is important to know why we use those libraries, to give you more insight, this is pretty much what help us to make an ajax call inside an action and return functions inside an action instead of just POJOS, search a little bit about 'thuck' functions and we will discusse it a little bit.